src/lib/containers/DiscussionSearchResult.tsx

```diff
@@ -35,51 +36,69 @@ const DiscussionSearchResult = (props: DiscussionSearchResultProps) => {
+            newMessageUrl = await getThreadMessageUrl(thread.messageKey, accessToken)
         }
-        setMessageUrl(await getMessage(someUrl.messageUrl))
+        setMessageUrl(await getMessage(newMessageUrl.messageUrl))
         setThreadBundle(thread)
+        setIsLoading(false)
     }
 
     useEffect(() => {
         getThreadOrReply()
     }, [])
 
-    const getUrl = (threadId: string, replyId: string, projectId: string) => {
+    const getUrl = (threadId: string, projectId: string, replyId?: string,) => {
         return `https://synapse.org/#!Synapse:${projectId}/discussion/threadId=${threadId}&replyId=${replyId}`
     }
 
     return (
         <div className='search-result-container'>
-            <Row>
-                <Col xs={1}>
-                    <img src={replyId ? ReplyIcon : ThreadIcon} alt={replyId ? 'reply-icon' : 'thread-icon'} className='search-result-icon' />
-                </Col>
-                <Col xs={11}>
-                    <Typography variant='headline3'>
-                        <a className='link' href={getUrl(threadBundle?.id!, replyBundle?.id!, threadBundle?.projectId!)}>{threadBundle?.title}</a>
-                    </Typography>
-                    <div className='truncated'>
-                        <Typography variant='body1'>
-                            {messageUrl}
+            {isLoading ?
+                <LoadingDiscussionSearchResult />
+                :
+                <Row>
+                    <Col xs={1}>
+                        {replyId ? <IconSvg options={{ icon: 'replyTwoTone' }} /> : <IconSvg options={{ icon: 'chatTwoTone' }} />}
+                    </Col>
+                    <Col xs={11}>
+                        <Typography variant='headline3'>
+                            <a className='link' href={getUrl(threadBundle?.id!, replyBundle?.id!, threadBundle?.projectId!)}>{threadBundle?.title}</a>
                         </Typography>
-                    </div>
-                    <div className='search-result-footer'>
-                        {replyId ? 'Reply' : 'Thread'} by {<UserCard size={SMALL_USER_CARD} ownerId={replyAuthor?.ownerId!} />} {moment(replyBundle?.createdOn).format('l LT')}
-                    </div>
-                </Col>
-            </Row>
+                        <div className='truncated'>
+                            <Typography variant='body1'>
+                                {messageUrl}
+                            </Typography>
+                        </div>
+                        <div className='search-result-footer'>
+                            {replyId ? 'Reply' : 'Thread'} by {<UserCard size={SMALL_USER_CARD} ownerId={replyAuthor?.ownerId!} />} {moment(replyBundle?.createdOn).format('l LT')}
+                        </div>
+                    </Col>
+                </Row>
+            }
+        </div>
+    )
+}
+
+export const LoadingDiscussionSearchResult: React.FunctionComponent = () => {
+    return (
```

Comment:
This should mirror the UI structure of a Discussion Search Result.  What does this look like today?

Reply:
Also, expected to see a list of these shown while search results are loading (instead of a spinner)